### ISSUE TITLE
No photos

### ISSUE THREAD

**Issue by mcfarnell** [leonspok/Irvue-Screensaver#11]
I installed this app, but it is not showing any photos. It is not downloading any photos. When I choose it as a my screensaver, it just shows a black screen instead of photos. What should I do?

**Comment by tjaadvd** [comment]
The developer told me 'there is a bug with combined channel. So, to make Irvue working again you need to choose some other channel and then relaunch it.' Does this fix your problem?

**Comment by jmunninghoff** [comment]
I'm having the same issue.  Not clear on what you mean by combined channel?

Thanks!

**Comment by jmunninghoff** [comment]
@tjaadvd I'm having the same issue. Not clear on what you mean by combined channel?

Thanks!

**Comment by tjaadvd** [comment]
I mean a channel like 'New & Featured':
<img width="272" alt="schermafbeelding 2018-09-01 om 11 38 17" src="https://user-images.githubusercontent.com/171732/44944689-8ecc5900-addb-11e8-9e20-209603b18707.png">

**Comment by mcfarnell** [comment]
No, I'm still having the issue. @tjaadvd I don't even have a menu like your screenshot. I'm on a mac, and can only change the screensaver settings in the system settings area? 
![image](https://user-images.githubusercontent.com/34324669/44952782-10080800-ae3d-11e8-8d6c-bbae260ae9f1.png)

**Comment by ckhatton** [comment]
@tjaadvd is confused and thinks you are talking about the app, whereas the issue is about the screensaver.

**Comment by tjaadvd** [comment]
@ckhatton Yes you are right.
